Tests/CommandsTests/RunToolTests.swift

```diff
@@ -19,6 +19,18 @@ final class RunToolTests: XCTestCase {
     private func execute(_ args: [String], packagePath: AbsolutePath? = nil) throws -> String {
         return try SwiftPMProduct.SwiftRun.execute(args, packagePath: packagePath, printIfError: true)
     }
+    private func executeTrash(_ args: [String], packagePath: AbsolutePath? = nil) throws {
+        do {
+        _ = try SwiftPMProduct.SwiftRun.execute(args, packagePath: packagePath, printIfError: false)
+        } catch SwiftPMProductError.executionFailure(_, _, _) {
```

Comment:
I think this can be `catch SwiftPMProductError.executionFailure` or `catch SwiftPMProductError.executionFailure(_)` but not sure.